Puppetfile

```diff
@@ -2,7 +2,7 @@ forge 'https://forgeapi.puppetlabs.com'
 
 # Dependencies
 mod 'puppetlabs/mysql',         '>= 4.0.0'
-mod 'puppetlabs/postgresql',    '>= 4.8.0'
+mod 'puppetlabs/postgresql',    :git => 'https://github.com/puppetlabs/puppetlabs-postgresql'
```

Comment:
Could you add a comment linking to https://tickets.puppetlabs.com/browse/MODULES-6963 so we know why it's there?